DESCRIPTION

```diff
@@ -17,10 +17,11 @@ Depends: R (>= 3.4.0)
 Imports:
     Rcpp (>= 0.12.16),
     methods,
+    broom,
```

Comment:
Is tidy ever used in the actual package? If only used in tests, can make broom a suggests.

Reply:
Yes it is, in a couple of places (e.g. `commarobust`).